interface/super/edit_list.php

```diff
@@ -614,31 +616,21 @@ function writeITLine($it_array) {
 <html>
 
 <head>
-<?php html_header_show();?>
-
-<!-- supporting javascript code -->
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-2-2/index.js"></script>
-
-<link rel="stylesheet" href='<?php  echo $css_header ?>' type='text/css'>
+<?php echo Header::setupHeader('select2'); ?>
 <title><?php  xl('List Editor','e'); ?></title>
-
 <style>
-tr.head   { font-size:10pt; background-color:#cccccc; text-align:center; }
-tr.detail { font-size:10pt; }
-td        { font-size:10pt; }
-input     { font-size:10pt; }
-a, a:visited, a:hover { color:#0000cc; }
 .optcell  { }
 .optin    { background-color:transparent; }
 .help     { cursor:help; }
 .translation { color:green; }
 </style>
-
-<script type="text/javascript" src="../../library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
 <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jscolor-1-4-5/jscolor.js"></script>
```

Comment:
(note it's already in the codebase since I think we had to keep the old one for the legacy code)